src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowsCache.cs

```diff
@@ -66,8 +74,47 @@ public bool TryGetValue(string key, out SKImage? image)
 
 	public bool Remove(string key)
 	{
-		System.Diagnostics.Debug.WriteLine($"[ShadowsCache] Remove => key: {key}");
-
 		return _shadowsCache.TryRemove(key, out var _);
 	}
+	public async void CleanupIfNeeded()
+	{
+		if ((DateTime.UtcNow - _lastCleanupTime).TotalSeconds > CleanupInterval)
+		{
+			try
+			{
+				await CleanupAsync();
```

Comment:
Since we will not do anything else here, we don't need to return to the caller thread.

```suggestion
				await CleanupAsync().ConfigureAwait(false);
```